test/unittest/transport/SharedMemTests.cpp

```diff
@@ -364,110 +357,6 @@ TEST_F(SHMRingBuffer, listeners_register_unregister)
     listener2->pop();
 }
 
-TEST_P(SHMRingBufferMultiThread, multiple_writers_listeners)
```

Comment:
Why is this test being removed?

Reply:
We decided to remove it. It is commented from the beginning and never works